sdk/confidentialledger/Azure.Security.CodeTransparency/README.md

```diff
@@ -45,18 +45,32 @@ To submit the signature, use the following code:
 CodeTransparencyClient client = new(new Uri("https://<< service name >>.confidential-ledger.azure.com"), null);
 FileStream fileStream = File.OpenRead("signature.cose");
 BinaryData content = BinaryData.FromStream(fileStream);
-Operation<GetOperationResult> operation = await client.CreateEntryAsync(content);
-Response<GetOperationResult> operationResult = await operation.WaitForCompletionAsync();
-Console.WriteLine($"The entry id to use to get the entry and receipt is {{{operationResult.Value.EntryId}}}");
-Response<BinaryData> signatureWithReceiptResponse = await client.GetEntryAsync(operationResult.Value.EntryId, true);
+Operation<BinaryData> operation = await client.CreateEntryAsync(content);
+Response<BinaryData> operationResult = await operation.WaitForCompletionAsync();
+
+string entryId = string.Empty;
+CborReader cborReader = new CborReader(operationResult.Value);
+cborReader.ReadStartMap();
+while (cborReader.PeekState() != CborReaderState.EndMap)
+{
+    string key = cborReader.ReadTextString();
+    if (key == "EntryId")
+    {
+        entryId = cborReader.ReadTextString();
+    }
+    else
+        cborReader.SkipValue();
+}
+Console.WriteLine($"The entry id to use to get the entry and receipt is {{{entryId}}}");
+Response<BinaryData> signatureWithReceiptResponse = await client.GetEntryAsync(entryId);
```

Comment:
Entry is just a receipt, and example should show GetEntryStatementAsync instead which will have the transparent statement (signature + receipt)